Sources/BraveWallet/Crypto/Asset Details/NFTDetailView.swift

```diff
@@ -0,0 +1,152 @@
+            .font(.title3.weight(.semibold))
+            .foregroundColor(Color(.braveLabel))
+          Text(nftDetailStore.nft.name)
+            .foregroundColor(Color(.secondaryBraveLabel))
+          Button(action: {
+            buySendSwapDestination = BuySendSwapDestination(
+              kind: .send,
+              initialToken: nftDetailStore.nft
+            )
+          }) {
+            Text(Strings.Wallet.nftDetailSendNFTButtonTitle)
+              .frame(maxWidth: .infinity)
+          }
+          .buttonStyle(BraveFilledButtonStyle(size: .large))
+        }
+        if let erc721MetaData = nftDetailStore.erc721MetaData {
+          if let description = erc721MetaData.description {
+            VStack(alignment: .leading, spacing: 8) {
+              Text(Strings.Wallet.nftDetailDescription)
+                .font(.headline.weight(.semibold))
+                .foregroundColor(Color(.braveLabel))
+              Text(description)
+                .foregroundColor(Color(.braveLabel))
+            }
+          }
+        }
+        VStack(spacing: 16) {
+          Group {
+            HStack {
+              Text(Strings.Wallet.nftDetailBlockchain)
+                .font(.headline.weight(.semibold))
+              Spacer()
+              Text(nftDetailStore.networkInfo.chainName)
+            }
+            HStack {
+              Text(Strings.Wallet.nftDetailTokenStandard)
+                .font(.headline.weight(.semibold))
+              Spacer()
+              Text(Strings.Wallet.nftDetailERC721)
+            }
+            HStack {
+              Text(Strings.Wallet.nftDetailTokenID)
+                .font(.headline.weight(.semibold))
+              Spacer()
+              Button(action: {
+                if let explorerURL = nftDetailStore.networkInfo.blockExplorerUrls.first {
+                  let baseURL = "\(explorerURL)/token/\(nftDetailStore.nft.contractAddress)"
+                  var nftURL = URL(string: baseURL)
+                  if let tokenId = Int(nftDetailStore.nft.tokenId.removingHexPrefix, radix: 16) {
+                    nftURL = URL(string: "\(baseURL)?a=\(tokenId)")
+                  }
+                  
+                  if let url = nftURL {
+                    openWalletURL?(url)
+                  }
+                }
+              }) {
+                if let tokenId = Int(nftDetailStore.nft.tokenId.removingHexPrefix, radix: 16) {
+                  Text(verbatim: "#\(tokenId)")
+                    .foregroundColor(Color(.braveBlurple))
+                } else {
+                  Text("\(nftDetailStore.nft.name) #\(nftDetailStore.nft.tokenId)")
+                    .foregroundColor(Color(.braveBlurple))
+                }
+              }
+            }
+          }
+          .foregroundColor(Color(.braveLabel))
+        }
+      }
+      .padding()
+    }
+    .onAppear {
+      nftDetailStore.fetchMetaData()
+    }
+    .navigationBarTitle(Strings.Wallet.nftDetailTitle)
```

Comment:
I think we want a different background color here. Maybe `.braveGroupedBackground`? cc @kylehickinson

Reply:
yep. will set .listBackgroundColor(Color(UIColor.braveGroupedBackground)) to be the same as AssetDetailView

Reply:
This one wasn't addressed in latest commit

Reply:
This should use `.background()` on every iOS version, this is not a `List` so it could break again in iOS 17 or later if behaviour changes.
The `.background()` modifier also needs `edgesIgnoreSafeArea(.vertical)` applied to the `Color` in the background so we fill the safe area:
<img width="488" alt="Screen Shot 2022-11-29 at 2 26 25 PM" src="https://user-images.githubusercontent.com/5314553/204629457-b7909039-2351-4e7e-9094-77e8266d4c92.png">